### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by T-Pham**
I just had so much spare time..

### CODE REVIEW

**Comment by toantran-ea** [comment]
👍  💯

**Comment by huydx** [comment]
Thanks~

File: README.md [hunk 1]
```diff
@@ -52,7 +51,7 @@ We are open to stack that are not listed too.
 
 ### What is the format of the job post?
 
-Every job post **MUST strictly** MUST be written in *English*. Furthermore, it must comply following rules:
+Every job post **MUST strictly** be written in *English*. Furthermore, it must comply the following rules:
```
Comment (khacanh) [inline README.md]: `comply with`?

Comment (T-Pham) [reply README.md]: Yeah, missed that one.

**Pull request merged**